modules/ingest-common/src/test/java/org/elasticsearch/ingest/common/ForEachProcessorTests.java

```diff
@@ -19,22 +19,22 @@
 
 package org.elasticsearch.ingest.common;
 
-import org.elasticsearch.ingest.CompoundProcessor;
-import org.elasticsearch.ingest.IngestDocument;
-import org.elasticsearch.ingest.Processor;
-import org.elasticsearch.ingest.TestProcessor;
-import org.elasticsearch.ingest.TestTemplateService;
-import org.elasticsearch.script.TemplateScript;
-import org.elasticsearch.test.ESTestCase;
-
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.Collections;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Locale;
 import java.util.Map;
+import org.elasticsearch.ingest.CompoundProcessor;
+import org.elasticsearch.ingest.IngestDocument;
+import org.elasticsearch.ingest.Processor;
+import org.elasticsearch.ingest.TestProcessor;
+import org.elasticsearch.ingest.TestTemplateService;
+import org.elasticsearch.script.TemplateScript;
+import org.elasticsearch.test.ESTestCase;
```

Comment:
Please change the import order in your IDE so that these sort before `java.util` and that would have led to these import orderings not changing. For example:

<img width="1168" alt="screen shot 2018-06-26 at 2 11 20 pm" src="https://user-images.githubusercontent.com/4744941/41931039-da3e873e-794a-11e8-99ec-0509c0fb20f3.png">

It's annoying to see these change in PR after PR based on different IDE settings.

Reply:
Done :), sorry about that